docs/examples/pythreejs-example.rst

```diff
@@ -0,0 +1,139 @@
+=================
+pythreejs Example
+=================
+
+Thebe can display output from pythreejs_. The examples are taken from the
+pythreejs documentation and are licensed BSD 3 Clause.
```

Comment:
A link to https://pythreejs.readthedocs.io/en/stable/examples/ or https://github.com/jupyter-widgets/pythreejs/tree/master/examples might be helpful here. 

Also, I would change the wording from
> ... and are licensed BSD 3 Clause.

to

> ... and are licensed under the BSD 3-Clause License.

to match the wording of the other Thebe docs pages and pythreejs's license.